lib/Bindings/Python/circt/dialects/_comb_ops_ext.py

```diff
@@ -1,11 +1,21 @@
 from circt.dialects import comb
-from circt.support import NamedValueOpView
+from circt.support import NamedValueOpView, get_value
 
-from mlir.ir import IntegerAttr, IntegerType
+from mlir.ir import IntegerAttr, IntegerType, Type
 
 from functools import reduce
 
 
+def infer_result_type(operands):
+  types = list(map(lambda arg: get_value(arg).type, operands))
+  if not types:
```

Comment:
Should this be `len(types) == 0`? Even if `not` does the same, I think the len is clearer.

Reply:
I think google style recommends `if not types`: https://google.github.io/styleguide/pyguide.html#2144-decision. But if it's more sane for an MLIR compiler developer, I can use `len(types) == 0`.

Reply:
I'm OK either way. It just confuses me every time, but that's just a guy who doesn't really know python.